docs/src/main/asciidoc/config.adoc

```diff
@@ -573,6 +573,55 @@ priority of 100.
 
 NOTE: This new converter also needs to be listed in a service file, i.e. `META-INF/services/org.eclipse.microprofile.config.spi.Converter`.
 
+== YAML for Configuration
+
+=== Add YAML-Config Support
+
+Sometimes you want to use YAML over properties for configuration.
+Since link:https://github.com/smallrye/smallrye-config[smallrye-config] brings support for YAML
+configuration, Quarkus supports this as well.
+
+First you will need to add the YAML-Extension to your POM-File.
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-config-yaml</artifactId>
+</dependency>
+----
+// TODO: Add the command to install this extension with maven.
+
+Now Quarkus can read YAML-Files.
+The config directories and priorities are the same as before.
+
+NOTE: Quarkus will choose a `application.properties` over a `application.yaml`.
+YAML-Files are just an alternative way to configure your application.
+You should decide and keep one configuration type to avoid errors.
+
+===== Configuration Example
+[source,properties]
+----
+# YAML supports comments
+greeting:
+  message: hello
+  name: quarkus
+  prizeAmount: 10
+  recipients:
+    - Jane
+    - John
```

Comment:
Maybe it would be better to add the datasource settings that was in the original issue?